cmd/fleetctl/package.go

```diff
@@ -220,11 +221,29 @@ func packageCommand() *cli.Command {
 				zlog.Logger = zerolog.Nop()
 			}
 
-			fmt.Println("Generating your osquery installer...")
-			path, err := buildFunc(opt)
+			const maxAttempts = 9 // see #5732
+			var (
+				attempts int
+				path     string
+				err      error
+			)
+			for attempts < maxAttempts {
+				attempts++
+
+				if attempts > 1 {
+					fmt.Printf("Generating your osquery installer [attempt %d/%d]...\n", attempts, maxAttempts)
```

Comment:
I really don't know why and don't want to go into the rabbit hole, but adding an extra newline here makes things work.

```suggestion
					fmt.Printf("Generating your osquery installer [attempt %d/%d]...\n\n", attempts, maxAttempts)
```

Reply:
Thanks! Merged your suggestion.